core/metacling/src/TClingCallbacks.cxx

```diff
@@ -288,8 +291,10 @@ bool TClingCallbacks::LookupObject(LookupResult &R, Scope *S) {
 
 bool TClingCallbacks::findInGlobalModuleIndex(DeclarationName Name, bool loadFirstMatchOnly /*=true*/)
 {
-   if (::getenv("ROOT_DISABLE_GMI"))
-      return false;
+   llvm::Optional<std::string> envUseGMI = llvm::sys::Process::GetEnv("ROOT_USE_GMI");
+   if (envUseGMI.hasValue())
+      if (!envUseGMI->empty() && !ROOT::FoundationUtils::ConvertEnvValueToBool(*envUseGMI))
```

Comment:
So `export ROOT_DISABLE_GMI=""` is the same as `export ROOT_DISABLE_GMI="false"`? (Just asking, might be worth a comment.)

Reply:
Will assert and convert to false.

Reply:
Sorry, I realized that I was too terse. In release mode setting ROOT_USE_GMI="" will behave as setting it to ROOT_USE_GMI="false". In debug mode it will assert as we cannot convert "" to bool.

I do not think a comment will help but I can issue a diagnostic in the conversion logic of that'd help.